### ISSUE TITLE
Data and Slide Show Missing

### ISSUE THREAD

**Issue by RamolaWeb** [utkarshsingh99/NIT-Website#13]
The data of the website is not visible and the slide show  is not also visible
@drtweety  @utkarshsingh99 @Abhinavlamba  @srajika256 @AnshuAkansha  Please Look into it

**Comment by utkarshsingh99** [comment]
Yes sir, we have tried remaking the site since there were a lot of discrepancies and design flaws in the previous one.
